### PR TITLE
run fullMIR tests in appveyor

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

File: src/fn_call.rs [hunk 1]
```diff
@@ -733,12 +733,19 @@ impl<'a, 'mir, 'tcx: 'mir + 'a> EvalContextExt<'tcx> for EvalContext<'a, 'mir, '
...
-            "std::sys::imp::c::::SetThreadStackGuarantee" => {
+            "std::sys::windows::c::::AddVectoredExceptionHandler" |
+            "std::sys::windows::c::::SetThreadStackGuarantee" => {
                 let usize = self.tcx.types.usize;
                 // any non zero value works for the stdlib. This is just used for stackoverflows anyway
                 self.write_scalar(dest, Scalar::from_u128(1), usize)?;
             },
+            "std::sys::windows::c::::GetModuleHandleW" |
+            "std::sys::windows::c::::GetProcAddress" |
+            "std::sys::windows::c::::SetLastError" => {
```
[the remark this answers is not in the document]
Comment (RalfJung) [reply src/fn_call.rs]: We do that for C ABI calls but this is not C ABI. It's some other ABI.

We should do the same thing, really, but I'd have to figure out which ABI this is first...

**Comment by RalfJung** [comment]
Argh, AppVeyor does not trigger...

File: src/fn_call.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (eddyb) [reply src/fn_call.rs]: No, you should do this *for all ABIs*. The ABI doesn't matter.
Only `tcx.is_foreign_item(def_id)` matters.

[the remark this answers is not in the document]
Comment (RalfJung) [reply src/fn_call.rs]: Ah, I did not know about `is_foreign_item`, thanks!

**Comment by RalfJung** [comment]
Dang, now it runs into thread-local storage stuff...

**Comment by RalfJung** [comment]
Wow I can't believe this worked :D

**Pull request merged**